sopel/test_tools.py

```diff
@@ -47,7 +47,9 @@ def __init__(self, nick, admin=False, owner=False):
         self.nick = nick
         self.user = "sopel"
 
-        self.channels = ["#channel"]
+        channel = sopel.tools.Identifier("#Sopel")
+        self.channels = {
+            channel: sopel.tools.target.Channel(channel)}
```

Comment:
What's up with this line break? The line wouldn't be that long without it.

Reply:
I think I was using a longer variable name than `channel` when I made it.